tasks/label_analysis.py

```diff
@@ -29,6 +31,8 @@
     SpecialLabelsEnum.CODECOV_ALL_LABELS_PLACEHOLDER.corresponding_label
 )
 
+GLOBAL_LEVEL_LABEL_IDX = 0
```

Comment:
Seems like this should go deeper in the report service or `shared` or something.  I saw `0` explicitly referenced elsewhere and should maybe use this constant instead